lib/collection/src/operations/types.rs

```diff
@@ -388,11 +388,39 @@ impl Default for ScrollRequestInternal {
     }
 }
 
+fn points_example() -> Vec<api::rest::Record> {
+    let mut payload_map_1 = Map::new();
+    payload_map_1.insert("city".to_string(), Value::String("London".to_string()));
+    payload_map_1.insert("color".to_string(), Value::String("green".to_string()));
+
+    let mut payload_map_2 = Map::new();
+    payload_map_2.insert("city".to_string(), Value::String("Paris".to_string()));
+    payload_map_2.insert("color".to_string(), Value::String("red".to_string()));
+
+    vec![
+        api::rest::Record {
+            id: PointIdType::NumId(43),
+            payload: Some(Payload(payload_map_1)),
+            vector: Some(VectorStruct::Single(vec![0.875, 0.140625, 0.897_6])),
+            shard_key: Some("region_1".into()),
+            order_value: Some(23.into()),
+        },
+        api::rest::Record {
+            id: PointIdType::NumId(44),
+            payload: Some(Payload(payload_map_2)),
+            vector: Some(VectorStruct::Single(vec![0.75, 0.640625, 0.8945])),
+            shard_key: Some("region_1".into()),
+            order_value: Some(23.into()),
```

Comment:
`order_value` is only present if we explicitly specify order-by. This is not a default behavior and query examples are not going that.